lib/migrate/sources/fs-migrations.js

```diff
@@ -80,7 +82,7 @@ class FsMigrations {
 
   getMigration(migration) {
     const absoluteDir = path.resolve(process.cwd(), migration.directory);
-    return require(path.join(absoluteDir, migration.file));
+    return import(path.join(absoluteDir, migration.file));
```

Comment:
what's the advantage of this change?

Reply:
require cannot import es modules, whereas import can.